types/capella/block.yaml

```diff
@@ -73,10 +73,6 @@ Capella:
         properties:
           execution_payload_header:
             $ref: './execution_payload.yaml#/Capella/ExecutionPayloadHeader'
-          bls_to_execution_changes:
```

Comment:
Why is this removed? Is this related to EIP-4844/deneb?

Reply:
this seems like maybe a branch out of date issue ,but does look weird...

Reply:
ok so this looks like a cleanup, it is in BeaconBlockBodyCommon, so it gets pulled into here when the `allOf` includes `BeaconBlockBodyCommon`